### ISSUE TITLE
Input descriptions do not show up in docs

### ISSUE THREAD

**Issue by roninjin10** [sachinraja/trpc-playground#37]
If I describe an input like this:

``` typescript
this.trpc.procedure
    .input(
      z
        .object({
          address: z.string.describe('Any valid ethereum address'),
        })
        .describe(
          'returns nft metadata',
        ),
    )
```

The address description shows up in the docs but not the object description.   (see metadata endpoint)
![image](https://user-images.githubusercontent.com/35039927/209195897-ca9e1973-7aae-445d-9333-9e7b1bfe0b86.png)

**Comment by MatsDK** [comment]
This seems to be an issue with [zod-to-ts](https://github.com/sachinraja/zod-to-ts). It seems like it is only rendering descriptions on `ZodObject` fields (https://github.com/sachinraja/zod-to-ts/blob/main/src/index.ts#L142-#L144).

**Comment by roninjin10** [comment]
@MatsDK looked into it and the issue is not zod-to-ts.   We are expected to be passing in the description here
<img width="582" alt="image" src="https://user-images.githubusercontent.com/35039927/209573298-960c9d83-b707-4f84-8708-74dd90320471.png">
The description comes from a util method that merges inputs (TIL trpc can define and merge more than 1 input object) .    The description never gets copied over.  I put up a pr adding it here: https://github.com/sachinraja/trpc-playground/pull/38

**Comment by roninjin10** [comment]
Update: Actually my theory doesn't make sense because I only use 1 input so that should be getting copied over 
<img width="481" alt="image" src="https://user-images.githubusercontent.com/35039927/209573573-85113087-10e5-4058-b0a6-fdc3e854e7d0.png">

I have time to continue to debug later this week

**Comment by MatsDK** [comment]
I created the `printTypeForDocs` function to solve this problem. After testing, it seems like it is working. The `createTypeAlias` is adding the description of the input above the the type declaration.
![image](https://user-images.githubusercontent.com/67562518/209574586-50860684-cdae-4c49-b4a3-d09505aa1f03.png)
This change is not published yet.

**Comment by MatsDK** [comment]
I don't know how we should handle multiple inputParser. What if there are multiple descriptions? Maybe something like #38 is necessary in that case.

**Comment by roninjin10** [comment]
ah yea good point about many.   The code I wrote only the last one is available.   The cleanest way to do it might be to hackily concat them together so make it look like

/** description 1 **/
propertyOnOne: string
anotherPropertyOnOne: number
/** description 2 **/
propertyOnTwo: Date
anotherPropertyOnTwo: boolean

**Comment by roninjin10** [comment]
Another option would be to make them a zod.union and see what zod prints

**Comment by sachinraja** [comment]
published in `v1.0.3`, please open another issue if you want this for multiple input parsers
